gamemodes/horde/entities/entities/obj_vj_horde_homing_lightning/init.lua

```diff
@@ -3,29 +3,33 @@ AddCSLuaFile( "cl_init.lua" )
 AddCSLuaFile( "shared.lua" )
 include( 'shared.lua' )
 
-function ENT:Initialize()
-	self:SetModel("models/props_junk/watermelon01_chunk02c.mdl")
-	self:SetMaterial("invis")
+function ENT:Initialize()	
+	local pb_vert = 2
+	local pb_hor = 2
+	self:SetModel("models/crossbow_bolt.mdl")
+	self:PhysicsInitBox( Vector(-pb_vert,-pb_hor,-pb_hor), Vector(pb_vert,pb_hor,pb_hor) )
```

Comment:
you can define these vectors outside of the function so you dont have to recreate them each time